server/admin/api.go

```diff
@@ -156,3 +158,21 @@ func (s *Server) apiAccounts(w http.ResponseWriter, _ *http.Request) {
 	}
 	writeJSON(w, accts)
 }
+
+// apiAccountInfo is the handler for the '/account/{account id}' API request.
+func (s *Server) apiAccountInfo(w http.ResponseWriter, r *http.Request) {
+	acct := strings.ToLower(chi.URLParam(r, accountNameKey))
```

Comment:
Seems like the ToLower call is not needed.  hex.DecodeString seems to be find with mixed case.